app/models/geo_entity.rb

```diff
@@ -15,9 +15,11 @@ class GeoEntity < ApplicationRecord
 
   has_many :country_citations, foreign_key: 'country_id'
 
-  scope :countries, -> { where.not(iso3: nil) }
+  scope :countries, -> { where.not(iso3: nil || 'GBL') }
```

Comment:
Is this Global?

Reply:
Yep - it's created as part of the Red List import task. Still unclear about what to do regarding that - it definitely needs fixing as one of the next tickets and I may even remove this change as part of that.